glpi/files/opt/startup.sh

```diff
@@ -35,6 +35,41 @@ do
     find "$dir" -type f -exec chmod u+rw {} \;
 done
 
+# Set ACL for www-data user on marketplace directory
+setfacl -m user:www-data:rwx,group:www-data:rwx "/var/www/glpi/marketplace"
+
+# forward logs files to stdout/stderr
+# see https://stackoverflow.com/a/63713129
+info_logs=(
+    "/var/glpi/logs/event.log"
+    "/var/glpi/logs/cron.log"
+    "/var/glpi/logs/mail.log"
+)
+error_logs=(
+    "/var/glpi/logs/php-errors.log"
+    "/var/glpi/logs/sql-errors.log"
+    "/var/glpi/logs/mail-errors.log"
+    "/var/glpi/logs/access-error.log"
+)
+# Create log files if they do not exist and set rights for www-data user
+all_logs=(
+    "${info_logs[@]}"
+    "${error_logs[@]}"
+)
+for log in "${all_logs[@]}"
+do
+    if [ ! -f $log ];
+    then
+        touch $log
+        chown www-data:www-data "$log"
+        chmod u+rw "$log"
+    fi
+done
+# info log files to stdout
+tail -F ${info_logs[@]} > /proc/1/fd/1 2>&1 &
```

Comment:
In fact, this should not be necessary, as the output of the init scripts are sent to the docker logs automatically.

```suggestion
tail -F ${info_logs[@]} > /proc/1/fd/1 &
```